tar5.4_tool.py

```diff
@@ -0,0 +1,42 @@
+"""
+    fanction that return thee min's lengh of the argument
+"""
+def find_min(*interleave):
```

Comment:
I think you should check if there is a python method that can simplify this function or even replace it.